templates/signup.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'app_public.html' %}
+
+
+{% block title %} Sign Up {% endblock %}
+{% block content %}
+	{% with messages = get_flashed_messages() %}
+		{% if messages %}
```

Comment:
It might make sense to be pull the flash to a parent context, or load it from a sub template, but you don't have to worry about that now.